src/Extension/CwpControllerExtension.php

```diff
@@ -160,6 +165,20 @@ protected function triggerBasicAuthProtection()
         }
     }
 
+    /**
+     * Get the current request, either from an Injector service or from the current controller
+     *
+     * @return HTTPRequest|null
```

Comment:
Implicit `nulls` are a bit grim. Also shouldn't this return docblock line be prepended with "mixed"?

Reply:
They are, but it's a standard we've adopted to follow =)